doc/remailers.rst

```diff
@@ -0,0 +1,125 @@
+Autocrypt and Mailing Lists
+====
+
+General Considerations
+----
+
+"Mailing lists" are a special use case of e-mail, where some address is managed
+not as an inbox that stores e-mail for retrieval, but instead forwards all
+messages to a configurable list of subscribers. For the case of end-to-end
+encryption, the introduction of a relaying agent brings up the question of what
+constitutes an "Endpoint".
+
+For users with a typical intuition of end-to-end encryption, the expected end
+points of end-to-end encrypted mailing lists will likely be the individual
+subscribers. This is very difficult to achieve on top of existing e-mail
+infrastructure, since it requires either negotiation between all endpoints, or
+use of highly experimental encryption techniques such as homomorphic or
+attribute-based encryption.
+
+A compromise solution is to encrypt from the sender only to the remailer, which
+then decrypts the message and individually encrypts to all subscribers. This
+method strictly violates end-to-end confidentiality, since the remailer can
+access to the forwarded plaintext. However, the remailer will only need to
+handle sensitive data transiently, which means messages are secure as long as
+the remailer is honest at the time of sending.
+
```

Comment:
This analysis doesn't take into account future key compromise.  And adversary who captures a message while the remailer is honest ("at time of sending") and then later steals the remailer's decryption key can violate the confidentiality of the message.

This can be ameliorated with subkey rotation and other schemes, but if we don't go into that here, we're handwaving away a legit concern.

Another way that i've seen this framed is to say that a an encrypted remailer-style mailing list with n subscribers has n+1 points of vulnerability for each message.

Reply:
Heh, I was kind of aware but didn't know how to put it nicely without being too negative about the whole thing. The n+1 thing does sound like a good framing